contracts/core/LensHub.sol

```diff
@@ -538,8 +545,13 @@ contract LensHub is ILensHub, LensNFTBase, VersionedInitializable, LensMultiStat
         whenNotPaused
     {
         bytes32[] memory dataHashes = new bytes32[](vars.datas.length);
+        // vars.data always must be same length to vars.followedFromProfileIds and vars.profileIds
```

Comment:
Its ensured that `profileIds.length == followModuleDatas.length` in `InterationLogic.sol` but we also need to ensure that are also equal to `followedFromProfileIds` length
Should be done after this loop to "fail as soon as possible" if needed